store/sql_team_store.go

```diff
@@ -692,6 +692,35 @@ func (s SqlTeamStore) GetTeamsUnreadForUser(teamId, userId string) StoreChannel
 	return storeChannel
 }
 
+func (s SqlTeamStore) GetTeamUnread(teamId string) StoreChannel {
```

Comment:
Can we rename this function `GetChannelUnreadsForTeam`? Since that's really what it's doing